### ISSUE TITLE
Better spacing or more controls over CSS for web UI

### ISSUE THREAD

**Issue by tagoad** [astronomer/ap-airflow#457]
I am not sure if this is the correct place for this request, so let me know if i need to move it somewhere better.
I recently started using tags in our Astronomer instance and while they are very useful, the way the UI is currently styled makes it feel very busy and cramped. I tried to find a way to just override some of the styling, which seems possible in base airflow but not in astronomer. Can we get some better styling for the DAG list, or atleast a better way to apply styles ourselves? Here's an example of my concern.

without styling
![image](https://user-images.githubusercontent.com/43684949/162005535-de4a1cbe-c993-4cc2-9213-3704e110a4e4.png)

Changing the tags font-size to .75em and the DAG name font-size to 1.25em. I feel like this greatly improves the ability to quickly skim the names while keeping the tags readable.
![image](https://user-images.githubusercontent.com/43684949/162005437-fc876849-4b28-4093-bac5-65490260a82e.png)

If there is a way to do this and i just missed it, please let me know.

**Comment by tagoad** [comment]
Im guessing by the lack of response that this is not the correct place to post this...

**Comment by bbovenzi** [comment]
Yes, that is something we can update. Let me see if its an easy css change.

**Comment by blag** [comment]
@tagoad Apologies for the delayed response. I don't think this is a great place to report issues like this, but I also don't know of a better place to do so. Let me/us figure that out and get back to you. No need to move this someplace else or open another issue elsewhere at this point though.

Regardless of where it was posted, thank you for giving us some feedback and playing around with it a bit to get some CSS changes with accompanying screenshots. I would agree that your changes make the DAG names stand out just a bit more while keeping the tags readable.

I think we'll keep this open for now until it's fixed and deployed, or until we figure out a better place to report issues like this.

**Comment by tagoad** [comment]
Awesome, thank you!
